app/plugins.js

```diff
@@ -307,6 +311,13 @@ function requirePlugins() {
 
       // populate the name for internal errors here
       mod._name = basename(path);
+      try {
+        // eslint-disable-next-line import/no-dynamic-require
+        mod._version = require(resolve(path, 'package.json')).version;
```

Comment:
yes I think it is not an issue, I measure it in renderer with 3 plugins without requiring `package.json`: 8ms.
With requiring: 12ms.

I don't want to add asynchronous complexity here and in the renderer for such a little gain.